app/src/main/java/com/example/returnpals/ScheduleReturn.kt

```diff
@@ -15,69 +25,134 @@ import java.time.LocalDate
  * https://kotlinlang.org/docs/extensions.html#extension-functions
  */
 
-/**
- * This object is extended with ui functions defined in `com.example.returnpals.composetools`.
- * It's used as a namespace, grouping together features related to the "Schedule a Return" action.
- */
-object ScheduleReturn {
+/////////////////////////////////////////////////////////////////////////////
+// PUBLIC API
+////////////////////
 
-    @RequiresApi(Build.VERSION_CODES.O)
-    class ViewModel(init: PickupInfo) : androidx.lifecycle.ViewModel() {
+object ScheduleReturn {}
 
-        private val idManager: IdManager = IdManager()
-        private val _pickup: MutableStateFlow<PickupInfo>
-        val pickup: StateFlow<PickupInfo>
+@Composable
+fun ScheduleReturn(
+    navController: NavController,
+    pickupInfo: PickupInfo,
```

Comment:
This looks good as the ScheduleReturn State which handles the return process. state changes when the pickup info changes.